action.yml

```
@@ -7,6 +7,10 @@ inputs:
   path:
     description: 'The file or folder to be cached'
     required: true
+  copy-strategy:
```

Would generalize and it could be move, symlink, etc

```suggestion
  strategy:
```

---

made it more explicit since this is a caching action, could be confused with some other strategy type. but will change